.isort.cfg

```diff
@@ -1,6 +1,9 @@
 [settings]
 line_length=110
-known_third_party=twisted,migrate,sqlalchemy,ldap3,klein,future,requests,MySQLdb,zope,coverage,autobahn,pkg_resources,jinja2,mock,dateutil,sphinx,setuptools
+known_future_library=future
+known_twisted=twisted,zope,autobahn,klein
+known_third_party=migrate,sqlalchemy,ldap3,txrequests,requests,MySQLdb,coverage,jinja2,mock,dateutil,sphinx,setuptools
```

Comment:
I'd put `mock` into a separate group and place it *after* `_thirdparty`.

After that I am *happy*! :smile: